ethcore/src/verification/queue/mod.rs

```diff
@@ -202,8 +202,8 @@ struct Verification<K: Kind> {
 	verifying: Mutex<VecDeque<Verifying<K>>>,
 	verified: Mutex<VecDeque<K::Verified>>,
 	bad: Mutex<HashSet<H256>>,
-	more_to_verify: SMutex<()>,
-	empty: SMutex<()>,
+	more_to_verify: Mutex<()>,
```

Comment:
It seems like these locks were previously added so that they could be used with `std::sync::Condvar`. Previously the `Condvar` waited on `unverified`. Do you think we could remove these?

Reply:
Good question! Did that at first, but then I saw [another PR](https://github.com/paritytech/parity-ethereum/pull/3903/files?utf8=%E2%9C%93&diff=split&w=1#diff-afe0a092320d9405848cfdc98328afefR687) using `more_to_verify`, and it wasn't obvious if they can be safely removed. Anyway, it seems to work w/o them, so I'm going to push another commit with the removal.